src/tests/multihost/alltests/test_krb_access_provider.py

```diff
@@ -0,0 +1,257 @@
+"""Automation for Krb access provider tests ported from bash
+
+"""
+import pytest
+from sssd.testlib.common.expect import pexpect_ssh
+from sssd.testlib.common.utils import sssdTools
+from constants import ds_instance_name
+
+
+@pytest.fixture(scope='class')
+def custom_setup(session_multihost, setup_sssd_krb, create_posix_usersgroups, krb_connection_timeout):
+    """ Added neccessary sssd domain parameters """
+    tools = sssdTools(session_multihost.client[0])
+    sssd_params = {'sbus_timeout': 30,
+                   'services': "nss, pam",
+                   'config_file_version': 2}
+    tools.sssd_conf('sssd', sssd_params)
+    domain_section = f'domain/{ds_instance_name}'
+    domain_params = {'access_provider': 'krb5',
+                     'use_fully_qualified_names': 'False',
+                     'override_homedir': '/home/%u'}
+    tools.sssd_conf(domain_section, domain_params)
+    tools.clear_sssd_cache()
+
+    cmds = ["mkdir -p /home/foo3",
+            "chown foo3: /home/foo3",
+            "chmod 700 /home/foo3",
+            "truncate -s 0 /home/foo3/.k5login",
+            "chown foo3: /home/foo3/.k5login",
+            "restorecon -v /home/foo3/.k5login"]
+    for command in cmds:
+        cmd = session_multihost.client[0].run_command(command, raiseonerr=False)
+        assert cmd.returncode == 0, f'{command} did not execute successfully'
+
+
```

Comment:
Please add the marker for tier 2 for now so it is included in runs.
@pytest.mark.tier2